### DISCUSSION TITLE
Notification can be in "hidden items"

### DISCUSSION THREAD

**Discussion by basepi** [community/community#7370] [Pull Requests]
I use github.com/notifications heavily. Today I got a notification based on a mention. However, when I opened the notification, I was not taken to the mention. Nor could `cmd-f` find the mention. Turns out the mention was inside the hidden items, which I actually had to expand multiple times before the mention was visible on the page:

![image](https://user-images.githubusercontent.com/702318/140535298-be24195c-3ec2-447b-89a9-73f6f3e28035.png)

Hiding items to improve load times is a great feature. But if I receive a notification due to a mention, that mention should always load.

**Comment by terminalmage** [comment]
Fixing this seems like a no-brainer to me.

**Reply by nmagedman** [reply]
Two related scenarios: I want to link to a comment, so I click on its 3-dots menu and choose "Copy link".  That link stops working if the conversation is either hidden or resolved.

**Reply by MikhailRyazanov** [reply]
I've also just encountered that our link checker complains about a missing anchor ``#issuecomment-...`` on a webpage for an URL that links from our project documentation to a particular comment. Because the comment is now “hidden” and cannot be accessed without user's active participation. Moreover, even after clicking on “Load more” the focus is not moved to the linked comment! Dear GitHub, please don't aggravate [link rot](https://en.wikipedia.org/wiki/Link_rot)! If it's difficult to handle such situations automatically, please at least implement a URL parameter (like ``?expand_all``) that will show the full page so that we can use such URLs explicitly.

**Comment by bjchentc** [comment]
Is it possible to have the conversations in different pages from the push/system information? Every time when I try to browse the conversations, they could be in the hidden items, but sometimes the hidden items are really too many to easily find out what we truly wanna see in the PR.

**Comment by Audiopolis** [comment]
And how about this: if there are 99 resolved conversations and 1 unresolved conversation, the unresolved conversation gets precedence

**Reply by bjchentc** [reply]
It sounds pretty cool coz it'd be much easier to keep eyes on those unresolved ones. However, sometimes we'd probably need to look back to some previous modifications and discussions for ensuring the next step is correct. Maybe it could be configured personally in the settings?
